modules/alloydb/variables.tf

```
@@ -245,21 +245,33 @@ variable "name" {
 }
 
 variable "network_config" {
-  description = "Network configuration for cluster and instance. Only one between cluster_network_config and cluster_psc_config can be used."
+  description = "Network configuration for cluster and instance. Only one between psa_config and psc_config can be used."
   type = object({
-    network                      = string
-    allocated_ip_range           = optional(string, null)
-    authorized_external_networks = optional(list(string), null)
-    enable_public_ip             = optional(bool, false)
+    psa_config = optional(object({
+      network                      = optional(string, null)
+      allocated_ip_range           = optional(string, null)
+      authorized_external_networks = optional(list(string), null)
+      enable_public_ip             = optional(bool, false)
+    }), null)
```

```suggestion
    psa_config = optional(object({
      network                      = optional(string)
      allocated_ip_range           = optional(string)
      authorized_external_networks = optional(list(string), [])
      enable_public_ip             = optional(bool, false)
    }))
```
`null` is implied, if skipped in the optional